frontend/src/metabase/visualizations/echarts/cartesian/model/dataset.ts

```diff
@@ -363,13 +368,15 @@ function getStackedValueTransformFunction(
         }, 0);
       const rawTotal = rawBelowTotal + getNumberOrZero(datum[seriesDataKey]);
 
+      const transformedRawBelowTotal = valueTransform(rawBelowTotal) ?? 0;
```

Comment:
Would be better on line 377 so it's underneath the comment explaining that step.